libraries/rush-lib/scripts/copyEmptyModules.js

```diff
@@ -50,12 +50,12 @@ module.exports = {
       const folderItems = await FileSystem.readFolderItemsAsync(
         relativeFolderPath ? `${jsInFolderPath}/${relativeFolderPath}` : jsInFolderPath
       );
-      for (const folderItem of folderItems) {
+      await Async.forEachAsync(folderItems, async (folderItem) => {
         const itemName = folderItem.name;
         const relativeItemPath = relativeFolderPath ? `${relativeFolderPath}/${itemName}` : itemName;
 
         if (folderItem.isDirectory()) {
-          await searchAsync(relativeItemPath);
+          return searchAsync(relativeItemPath);
```

Comment:
Our repo generally likes to await the promises inline rather than return them from async functions because it provides a more informative callstack if it does fail.